Modules/Map/QuestieMap.lua

```diff
@@ -329,7 +329,6 @@ function QuestieMap.ProcessQueue()
         if minimapDrawCall[2]._needsUnload then
             minimapDrawCall[2]:Unload()
         end
-    end
```

Comment:
This `end` still needs to be added back.

Reply:
I'm not totally sure why this was not fixed last time. Anyways, it should be fixed now.